src/Core/Domain/Entities/Users/User.cs

```diff
@@ -21,6 +21,8 @@ public User()
         public bool IsActive { get; set; }
 
         public DateTimeOffset? LastLoginDate { get; set; }
+        public string? RefreshToken { get; set; }
```

Comment:
Rather than adding RefreshToken to user entity, it's better to create another entity with the following properties:

    public string Token { get; set; }
    public DateTime Created { get; set; }
    public DateTime Updated { get; set; }

Reply:
then I have to add a UserId column in this new table too, so as to differentiate

Reply:
exactly, these properties + UserId.